src/meta/src/hummock/metrics_utils.rs

```diff
@@ -56,27 +56,31 @@ pub fn trigger_sst_stat(
         });
         level_sst_size / 1024
     };
-    for (idx, level_handler) in enumerate(compact_status.level_handlers.iter()) {
+
+    for idx in 0..current_version.levels.len() {
```

Comment:
After #3111, L0 is separated from `levels` so the total number of levels is `current_version.levels.len() + 1`. You can add a `num_levels` method in [HummockVersionExt](https://github.com/singularity-data/risingwave/blob/6765d2b42465a46402d4babc0f058ba2a91240e0/src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs#L291) to avoid further misuage.

Reply:
`compact_status.level_handlers` includes level0 and other levels while `current_version.levels` only includes level1-n :(.